docs/source/links/rest.md

```diff
@@ -0,0 +1,148 @@
+---
+title: apollo-link-rest
+description: Call your REST APIs inside your graphQL queries.
+---
+
+[**Read the announcement post! ( to be pusblished soon )
+🎉**](https://dev-blog.apollodata.com/)
+
+If you have a REST API, you can now query your endpoints using GraphQL and have all your data managed by Apollo Client. There is no need to write a wrapper around your REST API.
+
+<h2 id="start">Quick start</h2>
```

Comment:
Quick start should show a full example from setup to query (see state link docs for an example). You should assume that the person reading does not have experience with Apollo. REST link is a great way for developers who don't have a GraphQL server to start experimenting with Apollo, so it's really important that this example is complete.